src/contrib/mlir/backend/cpu/cpu_backend.cpp

```diff
@@ -194,7 +194,8 @@ void MLIRCPUBackend::lowerNgDialect()
 void MLIRCPUBackend::lowerStandardDialect()
 {
     mlir::PassManager pm(&m_context);
-    pm.addPass(mlir::createLowerToLLVMPass());
+    pm.addPass(mlir::createLowerToLLVMPass(
+        false /*useAlloca*/, false /*useBarePtrCallConv*/, true /*emitCWrappers*/));
```

Comment:
Do you think we could use the default calling convention (flattened memref descriptors) instead to avoid the extra function call? It could be a follow-up patch but at least here we have to make sure the these can be easily integrated with https://github.com/NervanaSystems/ngraph/pull/4336. Do you foresee any issue?